v2/samples/dbformysql/v1api/v1_user_aad.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: dbformysql.azure.com/v1
+kind: User
+metadata:
+  name: sampleaaduser
+  namespace: default
+spec:
+  owner:
+    name: samplemysql
+  # Specify a list of server-level privileges. Privileges
+  # include the following: RELOAD, PROCESS, SHOW DATABASES,
+  # REPLICATION SLAVE, REPLICATION CLIENT, CREATE USER
+  privileges:
+    - PROCESS
+    - CREATE USER
+  databasePrivileges:
+    mysqldatabase-sample:
+      # Privileges include the following:
+      # SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, REFERENCES, INDEX,
+      # ALTER, CREATE TEMPORARY TABLES, LOCK TABLES, EXECUTE, CREATE VIEW,
+      # SHOW VIEW, CREATE ROUTINE, ALTER ROUTINE, EVENT, TRIGGER
```

Comment:
Saying "include" implies there are other privileges that might be used. Is this canonical or just representative?

Reply:
It's representative. The full set is here I think: https://dev.mysql.com/doc/refman/5.7/en/privileges-provided.html

It depends on the version of MySQL, and may also be expandable through MySQL addons/plugins (I'm not enough of an expert in the space to know).  We don't do validation at the operator level on these, we let MySQL perform validation. Being exhaustive here is difficult, but these are the common ones.